bundle/android/strings.xml

```diff
@@ -227,9 +227,9 @@
     <string name="gutenberg_native_jetpack_powered" tools:ignore="UnusedResources">Jetpack powered</string>
     <string name="gutenberg_native_line_height" tools:ignore="UnusedResources">Line Height</string>
     <string name="gutenberg_native_link_inserted" tools:ignore="UnusedResources">Link inserted</string>
+    <string name="gutenberg_native_link_label" tools:ignore="UnusedResources">Link label</string>
```

Comment:
These changes are related to https://github.com/WordPress/gutenberg/pull/60047 so it's ok they're being updated here.